modin/engines/base/io/io.py

```diff
@@ -55,49 +49,61 @@ class BaseIO(object):
 
     @classmethod
     def from_non_pandas(cls, *args, **kwargs):
-        """Improve non-pandas object to an advanced and superior Modin DataFrame."""
+        """
+        Improve non-pandas object to an advanced Modin query compiler.
+
+        Parameters
+        ----------
+        *args : iterable
+            Positional arguments to be passed into `func`.
+        **kwargs : dict
+            Keyword arguments to be passed into `func`.
+
+        Notes
+        -----
+        Currently returns nothing.
```

Comment:
this is kinda moot point IMHO :)

Reply:
Changed to `None`.

Reply:
My point was this is kind of useless to state that it "currently returns nothing".
This function is by definition a point of extending, and it's irrelevant what it does in base implementation.

This note brings nothing useful and should be removed IMHO.

Reply:
Oh, i see now. Note was removed.